<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calanderWidget</class>
 <widget class="QWidget" name="calanderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1360</width>
     <height>780</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1360</width>
      <height>780</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{border-image: url(:/calander);}</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>1275</width>
       <height>591</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{background-color: rgba(0, 0, 0, 0);gridline-color: rgba(0, 0, 0, 50);}</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>15</number>
     </property>
     <property name="columnCount">
      <number>8</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>8:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>16:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>17:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>18:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>19:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>20:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>21:00</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>22:00</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mon.</string>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tue.</string>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Wed.</string>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Thur.</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fri.</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sat.</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sun.</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>8:00</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>9:00</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>10:00</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>11:00</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>12:00</string>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>13:00</string>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>14:00</string>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string>15:00</string>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>16:00</string>
      </property>
     </item>
     <item row="9" column="0">
      <property name="text">
       <string>17:00</string>
      </property>
     </item>
     <item row="10" column="0">
      <property name="text">
       <string>18:00</string>
      </property>
     </item>
     <item row="11" column="0">
      <property name="text">
       <string>19:00</string>
      </property>
     </item>
     <item row="12" column="0">
      <property name="text">
       <string>20:00</string>
      </property>
     </item>
     <item row="13" column="0">
      <property name="text">
       <string>21:00</string>
      </property>
     </item>
     <item row="14" column="0">
      <property name="text">
       <string>22:00</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="weekNum">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>17</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="closebtn">
     <property name="geometry">
      <rect>
       <x>1290</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minibtn">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="month">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>17</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="date">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>17</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nOfWeek">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>17</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="addbtn">
     <property name="geometry">
      <rect>
       <x>1185</x>
       <y>592</y>
       <width>115</width>
       <height>125</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="prevbtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="nextbtn">
     <property name="geometry">
      <rect>
       <x>1320</x>
       <y>170</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
